modules/ocf_desktop/manifests/pulse.pp

```diff
@@ -19,4 +19,11 @@
     require => Package['libasound2-plugins'],
   }
 
+  $pulse_sink = lookup({'name' => 'pulse_sink', 'default_value' => nil})
```

Comment:
Does this need an explicit default? I was under the impression it did not, although I have not worked that closely with hiera.